install_executorch.py

```diff
@@ -39,7 +39,7 @@ def clean():
     print("Done cleaning build artifacts.")
 
 
-VALID_PYBINDS = ["coreml", "mps", "xnnpack", "training"]
+VALID_PYBINDS = ["coreml", "mps", "xnnpack", "training", "openvino"]
```

Comment:
has this been tested?

Reply:
Yes, it seems to be working on our end.

Reply:
@jathu 

what's the recommended way after your recent refactoring/changes?

Reply:
I'm a bit confused, I don't see any pybind modules in this PR. If we don't have any pybindings, we don't need to add to this list